src/connectors/current-refined-values/connectCurrentRefinedValues.js

```diff
@@ -214,3 +218,14 @@ function clearRefinementFromState(state, refinement) {
 function clearRefinement(helper, refinement) {
   helper.setState(clearRefinementFromState(helper.state, refinement)).search();
 }
+
+function computeLabel(value) {
```

Comment:
not sure if it makes sense, but those symbols might be different in other alphabets etc, so this should be configurable (after we looked some up about it)

also some people might want to have this written in full "less than x more than y".

Might be an edge case though